datasets/us/fincen_special_measures/crawler.py

```diff
@@ -0,0 +1,114 @@
+from lxml import html
+import re
+from zavod import Context, helpers as h
+from normality import slugify
+from typing import Dict, Generator, cast
+from typing import List
+
+
+def convert_date(date_str: str) -> List[str]:
+    """Convert various date formats to 'YYYY-MM-DD'."""
+    # Regular expression to find dates in the format 'MM/DD/YYYY'
+    date_pattern = re.compile(r"(\d{1,2}/\d{1,2}/\d{4})")
+    date_pattern.findall(date_str)
+
+    formats = ["%m/%d/%Y"]  # 'MM/DD/YYYY' format
+    date = h.parse_date(date_str, formats)
+    return date
```

Comment:
so a couple of things

- the result of the regex findall wasn't being used, so we're getting one value (just a year) when multiple dates are available
- it's nice to compile regexes once at the top of the module, rather than each time they're used. I missed this earlier

```suggestion
from typing import List


REGEX_DATE = re.compile(r"(\d{1,2}/\d{1,2}/\d{4})")


def convert_date(date_str: str) -> List[str]:
    """Convert various date formats to 'YYYY-MM-DD'."""
    dates = date_pattern.findall(date_str)
    parsed = []
    formats = ["%m/%d/%Y"]  # 'MM/DD/YYYY' format
    for str in dates:
        parsed.extend(h.parse_date(str, formats)
    return parsed
```